internal/blobstore/blobstore.go

```diff
@@ -76,22 +76,65 @@ func NewContentChallengeResponse(chal *ContentChallenge, r io.ReadSeeker) (*Cont
 	}, nil
 }
 
+// Store stores data blobs in the configured store.
+type Store struct {
```

Comment:
Thanks. For some reason I struggled with making this read as I wanted and ended up with this. I think my desire was to leave all other code untouched and wrapping this interface in a struct meant not visiting every place the current Store struct is used as *Store and changing it to Store. I've visited each use and updated to Store.